lib/Runtime/Library/JavascriptError.cpp

```diff
@@ -108,6 +153,34 @@ namespace Js
 
 #undef NEW_ERROR
 
+    Var JavascriptAggregateError::NewAggregateErrorInstance(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+        ARGUMENTS(args, callInfo);
+        ScriptContext* scriptContext = function->GetScriptContext();
+        JavascriptAggregateError* pError = scriptContext->GetLibrary()->CreateAggregateError();
+        Var newTarget = args.GetNewTarget();
+        Var errors = args.Info.Count > 1 ? args[1] : scriptContext->GetLibrary()->GetUndefined();
+        Var message = args.Info.Count > 2 ? args[2] : scriptContext->GetLibrary()->GetUndefined();
+        Var options = args.Info.Count > 3 ? args[3] : scriptContext->GetLibrary()->GetUndefined();
+        return JavascriptAggregateError::NewInstance(function, pError, callInfo, newTarget, errors, message, options);
+    }
+
+    void JavascriptAggregateError::SetErrorsList(JavascriptAggregateError* pError, JavascriptAggregateErrorErrorsList* errorsList, ScriptContext* scriptContext)
+    {
+        JavascriptArray* array = scriptContext->GetLibrary()->CreateArray(errorsList->Count());
+        uint32 n = 0;
+        SList<Var, Recycler>::Iterator it = errorsList->GetIterator();
```

Comment:
Nit: Why not use JavascriptAggregateErrorErrorsList::Iterator as the type here? (would be the same I know)

Reply:
I have deleted JavascriptAggregateError..